src/Misc/UnpublishVSTSAgent.template.ps1

```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = 'Stop'
```

Comment:
We probably don't actually need this file anymore if we're going to only roll forward. Fine leaving it if you feel like we should have it for the moment though.

Reply:
I think we should leave it until we have a roll forward process defined.